src/simmate/toolkit/creators/structure/third_party/airss.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+import os
+import subprocess
+
+from simmate.toolkit import Composition, Structure
+from simmate.toolkit.creators.structure.base import StructureCreator
+
+
+class AirssStructure(StructureCreator):
+    """
+    Creates structures using the AIRSS software package.
```

Comment:
@jacksund This is looking exciting!!!  Let the competition begin!

Reply:
Yep! Time for some benchmarks :smile: